library/html2pdf/html2pdf.class.php

```diff
@@ -144,8 +144,12 @@ public function __construct($orientation = 'P', $format = 'A4', $langue='fr', $u
         $this->setTestIsImage(true);
         $this->setTestIsDeprecated(true);
 
-        // init the default font
-        $this->setDefaultFont(null);
+        //set 'dejavusans'. this font is supported a lot of languages - http://dejavu-fonts.org/wiki/Main_Page
+        $this->setDefaultFont('dejavusans');
```

Comment:
Wouldn't it make more sense to not hardcode this here and instead set it after create the html2pdf instance via $pdf->setDefaultFont() ?